stan/math/rev/mat/fun/matrix_exp_action_handler.hpp

```diff
@@ -5,10 +5,6 @@
 #include <stan/math/rev/core.hpp>
```

Comment:
This include should be removed and the entire file moved from `rev` to `prim`.  There's no use of `rev` in the file.

Reply:
removed.